### DISCUSSION TITLE
Do you have some plans, after 3.0.0 version is done, outside of opened issues?

### DISCUSSION THREAD

**Discussion by inson1** [clementwzk/OpenCalc#347] [General]
.

**Comment by clementwzk** [comment]
I wonder if I could add some more advanced features. I don't know exactly what or how yet, if you have any suggestions.

**Reply by inson1** [reply]
@Darkempire78 Outside of unit convertors, other types of calculators, I add every suggestion, I can think of, to issues
Btw #319 , #175, #301, #74 are for me the most important (in that order)
